mozjs-sys/build.rs

```diff
@@ -57,17 +57,40 @@ fn main() {
     // Used by mozjs downstream, don't remove.
     println!("cargo:outdir={}", build_dir.display());
 
-    if let Some(archive) = env::var_os("MOZJS_ARCHIVE") {
-        download_static_lib_binaries(&PathBuf::from(archive), &build_dir);
+    // Link to pre-built archive first if it exists.
+    let create_archive = env::var_os("MOZJS_CREATE_ARCHIVE").is_some();
+    let build_from_source = if env::var_os("MOZJS_FROM_SOURCE").is_some() {
+        println!("cargo:info=Environment variable MOZJS_FROM_SOURCE is set. Building from source directly.");
```

Comment:
`cargo:info` does not actually exist 😭. So we should either change this to `cargo:warning` or use normal prints, which will be available only when very verbose output is enabled. I prefer the letter.

Reply:
Okay I'll update to normal prints. It seems build script error will also show std output.